xml/System/String.xml

```diff
@@ -4493,7 +4493,7 @@ Depending on Why you want to call the `Copy` method, there are a number of alter
         <param name="action">A callback to initialize the string.</param>
         <summary>Creates a new string with a specific length and initializes it after creation by using the specified callback.</summary>
         <returns>The created string.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>The initial values of the span passed into <paramref name="action" /> are undefined. Therefore, it is caller's responsiblity to ensure every characters of the span are assigned to.</remarks>
```

Comment:
```suggestion
        <remarks>The initial content of the destination span passed to <paramref name="action" /> is undefined. Therefore, it is the delegate's responsiblity to ensure that every element of the span is assigned. Otherwise, the resulting string could contain random characters.</remarks>
```

Reply:
Thanks for the suggestion 🙂